### PR TITLE
feature(productdescription dropdown)

### PR DESCRIPTION
**Pull request by shravanipagilla**
# Description

Please provide a short summary explaining what this PR is about.

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by shpomp — APPROVED** [review]
👍🏽

**Comment by shpomp** [comment]
PR description and labels next time 🙏🏽

File: packages/client/src/components/ViewDropDown/ViewDropdown.component.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    </>
+  );
+};
+
+ViewDropdown.propTypes = {
+  options: PropTypes.string.isRequired,
+  label: PropTypes.string.isRequired,
+};
+
+export default ViewDropdown;
```
Comment (ShravyaGurmitkal) [inline packages/client/src/components/ViewDropDown/ViewDropdown.component.js:30]: Good

**Review by ShravyaGurmitkal — DISMISSED** [review]

**Review by RZhyvitskyi — APPROVED** [review]
Good with container

File: packages/client/src/containers/DropdownGroup/DropdownGroup.Container.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      </div>
+      <div className="dropdown-group">
+        <ViewDropdown options={Sizes} label="Size" />
+      </div>
+      <div className="dropdown-group">
+        <ViewDropdown options={Quantities} label="Quantity" />
+      </div>
+    </div>
+  );
+};
```
Comment (ShravyaGurmitkal) [inline packages/client/src/containers/DropdownGroup/DropdownGroup.Container.js:23]: Container looks good

**Review by ShravyaGurmitkal — APPROVED** [review]

**Pull request merged**